packages/cli/api-importers/openapi/openapi-v2-importer/src/3.1/OpenAPIConverter.ts

```diff
@@ -98,6 +100,36 @@ export class OpenAPIConverter extends AbstractConverter<OpenAPIConverterContext3
         };
     }
 
+    private removeXFernIgnores({ document }: { document: any }): any {
+        if (Array.isArray(document)) {
+            return document
+                .filter((item) => !(item != null && typeof item === "object" && item["x-fern-ignore"] === true))
+                .map((item) =>
+                    item != null && typeof item === "object" ? this.removeXFernIgnores({ document: item }) : item
+                );
+        } else if (document != null && typeof document === "object") {
+            return Object.fromEntries(
+                Object.entries(document)
+                    .filter(
+                        ([_, value]) =>
+                            !(
+                                value != null &&
+                                typeof value === "object" &&
+                                "x-fern-ignore" in value &&
+                                value["x-fern-ignore"] === true
```

Comment:
nit: `value != null && typeof value === "object" && "x-fern-ignore" in value && value["x-fern-ignore"] === true` we shouldn't duplicate this logic across arrays and objects, i could still see this logic living in `x-fern-ignore` and us instantiating an extension and calling `convert`